### ISSUE TITLE
Fix comments in MvxLocationWatcher WPF

### ISSUE THREAD

**Issue by Cheesebaron** [MvvmCross/MvvmCross#2911]
### Steps to reproduce :scroll:

1. Build sln
2. Look at output

### Expected behavior :thinking:
No false positive warnings with `error` should appear

### Actual behavior :bug:
Warnings are output with `See also error CS1056`. Makes it annoying to search for errors in build log

### Configuration :wrench:

**Version:** 6.x

**Comment by sharathtv89** [comment]
I will look into this issue.

**Comment by curtiscali** [comment]
Is this issue still up for grabs? I'm thinking of trying my hand at it soon.

**Comment by nickrandolph** [comment]
Yes I think if you look at the appveyor build output (or run MSBuild on solution) you will see these warnings

**Comment by niallermoran** [comment]
Building the solution doesn't seem to cause these warnings for me. I'd like to find an issue to start contributing.

ta

**Comment by Cheesebaron** [comment]
It is still there: https://ci.appveyor.com/project/MvvmCross/mvvmcross#L721

**Comment by fredeil** [comment]
I'm taking this issue. I will make a PR very soon 👍
